libs/nest/aws/src/lib/aws.module.ts

```diff
@@ -1,9 +1,16 @@
 import { Module } from '@nestjs/common'
 import { AwsService } from './aws.service'
+import { S3Client } from '@aws-sdk/client-s3'
 
 @Module({
   controllers: [],
-  providers: [AwsService],
-  exports: [AwsService],
+  providers: [
+    AwsService,
+    {
+      provide: S3Client,
+      useValue: new S3Client(),
+    },
+  ],
+  exports: [AwsService, S3Client],
```

Comment:
Do we need/want to export the S3Client in addition to the AwsService (which might be more appropriately named S3Service)?

Reply:
Whoops, no we dont, thought I had removed that, fixed now.
Promise to rename the service to S3Service in the next PR when I start placing it all over the place 😅